fighthealthinsurance/views.py

```diff
@@ -493,7 +494,8 @@ def form_valid(self, form):
         state = None
         zip_code = form.cleaned_data["zip"]
         if zip_code is not None and zip_code != "":
-            state = self.zip_engine.by_zip_code(form.cleaned_data["zip"]).state
+            print(f"Zip '{zip_code}'")
```

Comment:
Accidentally a print? 🙃

Reply:
Oh yeah was debugging that too :p lets drop it